src/Form/Builder/CheckboxFacetFormBuilder.php

```diff
@@ -4,13 +4,14 @@
 
 namespace Setono\SyliusMeilisearchPlugin\Form\Builder;
 
+use Setono\SyliusMeilisearchPlugin\Document\Metadata\Facet;
 use Symfony\Component\Form\Extension\Core\Type\CheckboxType;
 use Symfony\Component\Form\FormBuilderInterface;
 use function Symfony\Component\String\u;
 
 final class CheckboxFacetFormBuilder implements FacetFormBuilderInterface
 {
-    public function build(FormBuilderInterface $builder, string $name, array $values, array $stats = null): void
+    public function build(FormBuilderInterface $builder, string $name, array $values, Facet $facet, array $stats = null): void
```

Comment:
The name could be fetched from the `Facet` object now, couldn't it?

Reply:
Indeed! 👌